examples/tokens/ERC721.vy

```diff
@@ -62,18 +62,39 @@ ownerToNFTokenCount: uint256[address]
 # @dev Mapping from owner address to mapping of operator addresses.
 ownerToOperators: (bool[address])[address]
 
+# @dev Address of minter, who can mint a token
+minter: address
 
-# @dev Contract constructor. Per the Transfer event spec; during contract creation, any number of
-#      NFTs may be created and assigned without emitting Transfer.
+# @dev Mapping of interface id to bool about whether or not it's supported
+supportedInterfaces: bool[bytes32]
```

Comment:
Does `public` generate the correct getter `supportsInterface(_interfaceID: bytes32) -> bool`?

Reply:
hmm, yeah I guess "explicit is better than implicit", just seems like a waste of `public`'s utility

Reply:
>seems like a waste of public's utility

Yeah that's also true.